README.ja.md

```diff
@@ -44,8 +61,33 @@ $ vi config/config.secret.php   # tailor to your env
 $ COMPOSER_PROCESS_TIMEOUT=0 composer run
 ```
 
+Dockerの場合は以下コマンドで起動できます。
+
+```bash
+$ docker-compose up # 初回起動時はcompose installなどで時間がかかる
+```
+
 ブラウザで http://localhost:8888/index_dev.php/post/:post_number へアクセス。
 
+### 本番設定のサーバー起動
+
+Dockerの場合は[DockerHubのAutomated Build](https://hub.docker.com/r/kokuyouwind/esaba/)を使用して本番設定のApacheサーバを起動できます。
+
+`docker-compose.prod.yml` を使用する場合、`config`以下は名前付きボリュームになるため、`config.secret.php`を初回のみ設定することで、以降はコンテナを作り直しても設定が保持されます。
+
+```php
```

Comment:
this code is not php. Use `shell`.

Reply:
whoops

Reply:
You love PHP, too much.

Reply:
fixed https://github.com/standfirm/esaba/pull/2/commits/b3851fb9dbf3acbb08caade0c50302f22c222880